src/ui/profile_preferences.ui

```diff
@@ -134,6 +134,28 @@
         </widget>
        </item>
        <item row="1" column="0">
+        <widget class="QGroupBox" name="groupBox_encoding">
+         <property name="enabled">
+          <bool>true</bool>
+         </property>
+         <property name="title">
+          <string>Server Data Encoding</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_7">
+          <item>
+           <widget class="QComboBox" name="comboBox_encoding">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="toolTip">
```

Comment:
This needs to be reworded, as it is unlikely to be understood by the vast majority of the population. What are you trying to communicate here - I can help you reword it but I'm not sure what's the desired outcome is.